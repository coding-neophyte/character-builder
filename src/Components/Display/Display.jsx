import React from 'react'

export default function Display({catchPhrases}) {
    return (
        <div>
            {catchPhrases.map(phrase => <p key={phrase}>{phrase}</p>)}
        </div>
    )
}
