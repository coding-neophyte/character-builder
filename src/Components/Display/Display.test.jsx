import { render } from '@testing-library/react'
import Display from './Display'


it('should render catch phrases', () => {
    const {container} = render(<Display catchPhrases={['howdy', 'what it do', 'peace god']} />)


    expect(container).toMatchSnapshot();
})
