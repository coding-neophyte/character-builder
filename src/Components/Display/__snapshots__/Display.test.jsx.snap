// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render catch phrases 1`] = `
<div>
  <div>
    <p>
      howdy
    </p>
    <p>
      what it do
    </p>
    <p>
      peace god
    </p>
  </div>
</div>
`;
